test/blackbox/upload.bats

```diff
@@ -0,0 +1,45 @@
+load helpers
+
+function setup_file() {
+    # Download test data
+    skopeo --insecure-policy copy docker://public.ecr.aws/t0x7q1g8/centos:7 oci:${TEST_DATA_DIR}/zot-test:0.0.1
```

Comment:
Note skopeo installed via snap seems to need sudo, because `newuidmap`/`newgidmap` is not found by snap - must be a packaging issue or snap container issue.
Looks like skopeo installed via apt works fine without sudo.

So, as a side note related to the other make targets and https://github.com/project-zot/zot/issues/138, if we use the apt installed skopeo we may be able to remove some of the sudo usage in the Makefile.